OneSignalDevelopers/ngosi
# Center auth component on login page

Fixes : #22 

## Solution
I added some CSS properties that center the `Auth` component on the login page. 

## Testing
1. Start the supabase CLI, `supabase start` 
2. `yarn dev`, to start the dev server
3.  Navigate to `localhost:3000/signin` (signin page)  and observe the change made on the auth component


## Resources

###### Before adding styles
<img width="1435" alt="Screen Shot 2022-01-01 at 1 11 05 AM" src="https://user-images.githubusercontent.com/89279465/147845086-4fbe8239-1eed-413b-bcc9-acfe6a53b27c.png">

###### After adding styles
<img width="1433" alt="Screen Shot 2022-01-01 at 1 09 49 AM" src="https://user-images.githubusercontent.com/89279465/147845088-86e6814d-59fd-4ea0-9849-4852a4c00d4a.png">
